apoiase/app.py

```diff
@@ -20,17 +20,19 @@ def home():
     if response.status_code == 200:
         page = json.loads(response.text)
         users = page.get('supports', {}).get('users', [])
-        names = map(serializer, users)
-        names = [name for name in names if name]
+        users = filter(filter_private_supporters, users)
+        names = list(map(serializer, users))
         return jsonify(names)
 
 
-def serializer(users):
-    private=users.get('privateSupport')
-    if not private:
-        return dict(name=users.get('_id').get('name'))
+def filter_private_supporters(apoiase_users):
```

Comment:
This function kind of bugs me. Its syntax doesn't say what's the parameter for “filtering” users and its argument creates the expectation that it should receive a iterable of users (but it handles only a user per execution). What about a boolean syntax like?

```python
def is_private_user(user):
    private = user.get('privateSupport')
    return bool(private)
```